docs/governance/governance.md

```diff
@@ -23,149 +23,149 @@ The ODS Governance is designed with the following principles in mind:
 
-How content of the [ODS Specification](#operational-data-standard-ods-specification) may be changed and released as an official [ODS version](policies/change-management-versioning.md#ods-release).
+How content of the [TODS Specification](#operational-data-standard-tods-specification) may be changed and released as an official [TODS version](policies/change-management-versioning.md#tods-release).
 
-### ODS Documentation
+### TODS Documentation
 
-The text and website (and supporting scripts) which describe the [ODS Specification](#operational-data-standard-ods-specification) and its purpose and usage located on the [ODS Repository](#ods-repository).
+The text and website (and supporting scripts) which describe the [TODS Specification](#operational-data-standard-TODS-specification) and its purpose and usage located on the [TODS Repository](#tods-repository).
 
-### ODS Repository
+### TODS Repository
 
-The version control repository containing the [ODS Specification](#operational-data-standard-ods-specification) located at:
+The version control repository containing the [TODS Specification](#operational-data-standard-tods-specification) located at:
 <https://github.com/cal-itp/operational-data-standard>
 
-### ODS Repository Organization
+### TODS Repository Organization
 
-The version control organization which contains the [ODS Repository](#ods-repository).
+The version control organization which contains the [TODS Repository](#tods-repository).
 
 !!! warning "To be migrated"
 
-    The ODS Repository is currently located with the Cal-ITP GitHub organization but will be migrated as part of the process of transitioning management to MobilityData.
+    The TODS Repository is currently located with the Cal-ITP GitHub organization but will be migrated as part of the process of transitioning management to MobilityData.
 
-### ODS Tools
+### TODS Tools
 
-Any scripts or code released within the [ODS Repository Organization](#ods-repository-organization).
+Any scripts or code released within the [TODS Repository Organization](#tods-repository-organization).
 
-### ODS Use License
+### TODS Use License
 
-The [ODS Specification](#operational-data-standard-ods-specification) is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt) (code) and [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (sample data, specification, and documentation) as defined in LICENSES file in the [main repository](https://github.com/cal-itp/operational-data-standard).
+The [TODS Specification](#operational-data-standard-tods-specification) is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0.txt) (code) and [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (sample data, specification, and documentation) as defined in LICENSES file in the [main repository](https://github.com/cal-itp/operational-data-standard).
```

Comment:
nitpick: Should "LICENSES" be marked in code case, as it's the name of a file, and this is how we refer to the other *.md files mentioned on this page?

Reply:
Yes, agreed with that change

Reply:
Fixed in https://github.com/cal-itp/operational-data-standard/pull/78/commits/2235a5f95278d14b5db30e359a91a2f877b0e813